mungegithub/Dockerfile-publisher

```diff
@@ -15,7 +15,7 @@
 FROM google/debian:jessie
 MAINTAINER Chao Xu <xuchao@google.com>
 RUN apt-get update \
- && apt-get install -y -qq git=1:2.1.4-2.1+deb8u2 \
+ && apt-get install -y -qq git=1:2.1.4-2.1+deb8u4 \
```

Comment:
This is intentional. We depend quite a bit on git's behaviour and prefer to have it break than have wrong results.